crates/builder/src/tasks/submit.rs

```diff
@@ -43,10 +47,13 @@ impl SubmitTask {
             "pinging quincey for signature"
         );
 
+        let token = self.fetch_oauth_token().await?;
+
         let resp: reqwest::Response = self
             .client
             .post(self.config.quincey_url.as_ref())
             .json(sig_request)
+            .bearer_auth(token.access_token().secret())
```

Comment:
should we also address tx pool oath in the same PR?

we don't necessarily need to worry about efficiency of streamlining to one oath call for both tx pool & quincey within the example builder but would be a nice-to-have if it were simple

Reply:
currently we don't call the protected endpoints (`/get-order(s)`) so there isn't anything to update. Adding the functionality itself would be outside the scope of this